File: app/src/main/java/cz/vitskalicky/lepsirozvrh/bakaAPI/rozvrh/rozvrh3/Rozvrh3.java
package cz.vitskalicky.lepsirozvrh.bakaAPI.rozvrh.rozvrh3;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

/**
 * Rozvrh of the Bakaláři API v3
 */
@JsonIgnoreProperties(ignoreUnknown = true)
public class Rozvrh3 {
    Hour3[] hours;
    Day3[] days;
    Class3[] classes;
    Group3[] groups;
    Subject3[] subjects;
    Teacher3[] teachers;
    Room3[] rooms;
    Cycle3[] cycles;
}

File: app/src/main/java/cz/vitskalicky/lepsirozvrh/bakaAPI/rozvrh/rozvrh3/Rozvrh3.kt
package cz.vitskalicky.lepsirozvrh.bakaAPI.rozvrh.rozvrh3

/**
 * Rozvrh of the Bakaláři API v3
 */
data class Rozvrh3 (
    val hours: List<Hour3>,
    val days: List<Day3>,
    val classes: List<Class3>,
    val groups: List<Group3>,
    val subjects: List<Subject3>,
    val teachers: List<Teacher3>,
    val rooms: List<Room3>,
    val cycles: List<Cycle3>,
)